sphinx/builders/html/__init__.py

```diff
@@ -641,17 +641,17 @@ def gen_pages_from_extensions(self) -> None:
     def gen_additional_pages(self) -> None:
         # additional pages from conf.py
         for pagename, template in self.config.html_additional_pages.items():
-            logger.info(' ' + pagename, nonl=True)
+            logger.info(pagename + ' ', nonl=True)
```

Comment:
`@progress_message` decorator uses "after spacing" model, so logger output inside the decorator should use same model.

https://github.com/sphinx-doc/sphinx/blob/ae7c4cc3b87d270bd9089c8c145f569df2557b29/sphinx/util/__init__.py#L691-L707